.github/issue_labels/unknown_labels.txt

```diff
@@ -0,0 +1,127 @@
+1 min review
+10 min review
+20 min review
+30 min review
+5 min review
+:wave: team-triage
+@configuration-cache
+@core
+@dev-productivity
+@execution
+@ide-experience
+@idiomatic
+@support
+a:bug
+a:chore
+a:documentation
+a:feature
+a:investigation
+a:performance-improvement
+a:platformization
+a:regression
+affects-version:2.12
+affects-version:2.13
+affects-version:2.14
+affects-version:3.0
+affects-version:3.1
+affects-version:3.2
+affects-version:3.3
+affects-version:3.4
+affects-version:3.5
+affects-version:4.0
+affects-version:4.1
+affects-version:4.10
+affects-version:4.2
+affects-version:4.3
+affects-version:4.4
+affects-version:4.5
+affects-version:4.6
+affects-version:4.7
+affects-version:4.8
+affects-version:4.9
+affects-version:5.0
+affects-version:5.1
+affects-version:5.2
+affects-version:5.3
+affects-version:5.4
+affects-version:5.5
+affects-version:5.6
+affects-version:6.0
+affects-version:6.1
+affects-version:6.2
+affects-version:6.3
+affects-version:6.4
+affects-version:6.5
+affects-version:6.6
+affects-version:6.7
+affects-version:6.8
+affects-version:6.8.1
+affects-version:6.8.2
+affects-version:6.8.3
+affects-version:6.9
+affects-version:7.0
+affects-version:7.0.1
+affects-version:7.1
+affects-version:7.2
+affects-version:7.3
+affects-version:7.4
+affects-version:7.5
+affects-version:7.5.1
+affects-version:7.6
+affects-version:8.0
+affects-version:8.1
+affects-version:8.1.1
+affects-version:8.2
+affects-version:8.3
+android
+build-script-change
+cc-hackathon-2022
+closed:android
+closed:duplicate
+closed:eclipse
+closed:idea
+closed:invalid
+closed:kotlin
+closed:missing-dco
+closed:not-fixed
+closed:plugin
+closed:question
+closed:unreproducible
+cm-changes
+dev-prod:ignored-test
+epic
+flaky-fix-it
+from:contributor
+good first issue
+has:reproducer
+has:workaround
+is:blocked
+is:blocker
+p:lazy-migration
+pending:dco
+pending:pr-feedback
+pending:reproducer
+platform:bt-ge-build-cache
```

Comment:
🤔 probably it's a good time to revisit this platform:* labels
IIUC, nobody is using them.

However, we can use them as an aggregators for in: labels and assign platforms/owners